### ISSUE TITLE
Same display name parameters in settings

### ISSUE THREAD

**Issue by asand3r** [RocketChat/Rocket.Chat#11845]
### Description:

Where are two options:
Accounts_Default_User_Preferences_HideAvatars
Accounts_Default_User_Preferences_sidebarHideAvatars

But in settings they have one Display name - "Hide Avatars".

### Steps to reproduce:

1. Administration -> Accounts -> Default User Preferences

### Actual behavior:
![image](https://user-images.githubusercontent.com/13959303/44448482-0fc56c80-a5f4-11e8-8097-0c6eebac1980.png)

### Server Setup Information:

- Version of Rocket.Chat Server: 0.68.4
- Operating System: CentOS 7.5.1804
- Deployment Method: tar
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: 8.11.4 (lts)
- MongoDB Version: 4.0

**Comment by vynmera** [comment]
As the keys show, one hides the avatars in the sidebar, the other hides it in the messages view.

**Comment by asand3r** [comment]
@vynmera I get it. =) But in interface they looks same.

**Comment by vynmera** [comment]
@asand3r True, perhaps we need better i18n here... perhaps `Hide Avatars from Messages` and `Hide Avatars from Sidebar`?

**Comment by Shaverdoff** [comment]
i found another one miss i18n in 0.70.0
double "Placeholder for Password Login Field" in first section of Accounts
